aip/general/0121.md

```diff
@@ -81,6 +81,26 @@ In an API with a stateless protocol, the server has the responsibility for
 persisting data, which may be shared between multiple clients, while clients
 have sole responsibility and authority for maintaining the application state.
 
+### Cyclic References
+
+The relationship between resources, such as parent-child or
+[resource references][], **must** be representable via an acylic graph.
```

Comment:
Should we define acyclic graph? Link to a definition?

Reply:
Would the wikipedia article be ok? https://en.wikipedia.org/wiki/Directed_acyclic_graph.

I guess technically it's also directed, so I should fix that regardless.

Reply:
added.